File: app/src/main/java/org/lemonadestand/btb/features/login/models/Button.java
package org.lemonadestand.btb.features.login.models;

import com.google.gson.annotations.SerializedName;

public class Button {
    @SerializedName("position")
    String position;

    @SerializedName("x")
    int x;

    @SerializedName("y")
    int y;

    @SerializedName("append")
    String append;


    public void setPosition(String position) {
        this.position = position;
    }
    public String getPosition() {
        return position;
    }

    public void setX(int x) {
        this.x = x;
    }
    public int getX() {
        return x;
    }

    public void setY(int y) {
        this.y = y;
    }
    public int getY() {
        return y;
    }

    public void setAppend(String append) {
        this.append = append;
    }
    public String getAppend() {
        return append;
    }


}

File: app/src/main/java/org/lemonadestand/btb/models/Button.kt
package org.lemonadestand.btb.models

import com.google.gson.annotations.SerializedName

class Button {
	@SerializedName("position")
	var position: String? = null

	@SerializedName("x")
	var x: Int = 0

	@SerializedName("y")
	var y: Int = 0

	@SerializedName("append")
	var append: String? = null
}
